AlphaWallet/TokenScriptClient/Models/TokenIdOrigin.swift

```diff
@@ -21,18 +21,29 @@ struct TokenIdOrigin {
     }
 
     func extractValue(fromTokenId tokenId: TokenId) -> AssetInternalValue? {
-       let number: BigUInt = (bitmask & tokenId) >> bitShift
+        let number: BigUInt = (bitmask & tokenId) >> bitShift
```

Comment:
We'll have to do a bitmask and shift in `FunctionOrigin.extractValue()` like we do here in `TokenIdOrigin` here before `resultSubscribable.value = value` to change the value on the RHS.